server/tests/testarea.py

```diff
@@ -0,0 +1,39 @@
+from gqlComms import listHistograms, createHistogram, deleteHistogram,getHistogram,getHistograms
+import numpy as np
+import datetime
+import argparse
+
+def test_createHist():
+    NUM = 100
```

Comment:
By convention I would prefer if caps are global constants. If just declaring within a function keep it lower case, thanks